design/v8-bundle/project/styles_v7.css

```
@@ -0,0 +1,536 @@
+.oi-notif-body .reply {
+  margin-top: 12px;
+  font-size: 13px; line-height: 1.55; color: #f1ead9;
+  letter-spacing: -0.005em;
+  background: rgba(255,255,255,0.025);
+  border: 1px solid rgba(255,255,255,0.05);
+  border-radius: 12px;
+  padding: 12px 14px;
+  max-height: 280px;
+  overflow-y: auto;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)